docs/environment_variables.rst

```diff
@@ -0,0 +1,34 @@
+.. currentmodule:: astropy
+
+.. _environment_variables:
+
+*********************
+Environment variables
+*********************
+
+
+Runtime
+-------
+
+.. glossary::
+
+    ``XDG_CONFIG_HOME``
+        This environment variables control where configuration files are read
+        and written. Astropy will look for configuration files in
+        ``$XDG_CONFIG_HOME/astropy``.
+        See :ref:`astropy_config` for how to programmatically set or get the
+        location of the corresponding directory at runtime.
+
+    ``XDG_CACHE_HOME``
+        These environment variables control where data files are cached.
+        Astropy will cache files in ``$XDG_CACHE_HOME/astropy``.
+        See :ref:`utils-data` for how to programmatically set or get the
+        location of the corresponding directory at runtime.
+
+
+.. note::
+    ``XDG_CONFIG_HOME`` and ``XDG_CACHE_HOME`` come from a Linux-centric
+    specification
```

Comment:
We should emphasize that `astropy` has only adopted the names of these two environment variables from the specification and nothing else.

Reply:
the spec is specifically about UNIX-based systems though. I don't think it's a good idea to leave Windows users in the dark, especially because we *do* support them, so I don't agree with this take.

Reply:
The updated descriptions in the glossary are clear enough so that this note doesn't have to be edited.